src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
Got it, thanks.

On Tue, Jul 12, 2016 at 2:20 PM, jortiz16 <notifications@github.com> wrote:

> In src/edu/washington/escience/myria/coordinator/MasterCatalog.java
> <https://github.com/uwescience/myria/pull/830#discussion_r70524815>:
>
> > @@ -113,6 +114,7 @@
> >      + "    col_index INTEGER NOT NULL,\n"
> >      + "    col_name TEXT,\n"
> >      + "    col_type TEXT NOT NULL,\n"
> > +    + "    is_indexed INTEGER NOT NULL, \n"
>
> createIndexIfNotExistPostgres doesn't ignore the order from the indexRef
> list. The getIndexColumns in that method creates the sql string with the
> column names along with the order. The default is "ASC" if no order is
> specified.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/uwescience/myria/pull/830/files/4f8f7d56457c12dc57f9f0975fc979c49d08ae0e#r70524815>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAIBYXhkctLoEd8c5t95JMHh6dIrBjgoks5qVAUxgaJpZM4If5_N>
> .
>